### PR TITLE
Updated docker docs for location of data path

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated docker docs for location of data path

### CODE REVIEW

File: docs/Deploying-Pantheon/Migration-Docker.md [hunk 1]
```diff
@@ -36,10 +36,17 @@ was not available and the node key was always read from the data path.
...
-The [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) default is now `/opt/pantheon/database`. 
+The [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) default is now `/opt/pantheon`. 
 
 The [`--node-private-key-file`](../Reference/Pantheon-CLI-Syntax.md#node-private-key-file) default is 
-now `/opt/pantheon/database/key`. 
+now `/opt/pantheon/key`. 
+
+!!! important 
+    Do not mount a volume at the default data path (`/opt/pantheon`). Mounting a volume at the default 
+    data path stops the image from running.
```
Comment (Errorific) [inline docs/Deploying-Pantheon/Migration-Docker.md]: Might be better worded as "...path will interfere with Pantheon's operation and prevent it from safely launching."

File: docs/Getting-Started/Run-Docker-Image.md [hunk 2]
```diff
@@ -52,25 +52,32 @@ docker run -p <localportJSON-RPC>:8545 -p <localportWS>:8546 -p <localportP2P>:3
      
 ## Starting Pantheon 
 
+!!! important 
+    Do not mount a volume at the default data path (`/opt/pantheon`). Mounting a volume at the default 
+    data path stops the image from running.
```
Comment (Errorific) [inline docs/Getting-Started/Run-Docker-Image.md]: as per above comment

File: docs/Deploying-Pantheon/Migration-Docker.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Deploying-Pantheon/Migration-Docker.md]: Fixed

File: docs/Getting-Started/Run-Docker-Image.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Getting-Started/Run-Docker-Image.md]: Fixed

**Pull request merged**